src/envs/stick_point.py

```diff
@@ -320,6 +348,47 @@ def _get_tasks(self, num: int, num_point_lst: List[int],
                 "theta": theta,
                 "held": 0.0
             }
+            # Create the holder for the stick, sampling the position so that it
+            # is somewhere along the long dimension of the stick. To make sure
+            # that the problem is solvable, check that if the stick were
+            # grasped at the lowest reachable position, it would still be
+            # able to touch the highest point.
+            max_point_y = max(state_dict[p]["y"] for p in points)
+            necessary_reach = max_point_y - self.rz_y_ub
+            while True:
+                # Allow the stick to start in the middle of the holder.
+                x_offset = rng.uniform(-self._holder_width,
```

Comment:
why is it the case that in the videos, the holder can extend off the end of the stick? i would have thought it should be completely contained within the stick.